src/XML/src/FMI3/fmi3_xml_variable.c

```diff
@@ -1168,36 +1240,49 @@ int fmi3_xml_handle_IntXXVariable(fmi3_xml_parser_context_t* context, const char
             type = (fmi3_xml_int_type_props_t*)declaredType;
         }
         variable->type = &type->super;
+        /* TODO handle this buffer clean-up in a better way */
+        {
+            const char* tmp; /* unused */
+            fmi3_xml_get_attr_str(context, fmi3_xml_elmID_Int64, fmi_attr_id_start, 0, &tmp);
+        }
```

Comment:
We should handle this on all places before the ticket is done. We can save that for last but just keeping this thread open as reminder